terraform/aio/vm.tf

```
@@ -13,7 +13,7 @@ variable "aio_vm_name" {
 
 variable "aio_vm_image" {
   type    = string
-  default = "CentOS-stream8"
+  default = "bb8c0a34-533f-42fb-a49b-3461e677f3f6"
```

```suggestion
  default = "7efda0b1-e296-447f-896e-c066d0af5c53"
```

From: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4450001402/jobs/7815038031#step:8:56